atlasdb-cassandra/src/test/java/com/palantir/atlasdb/cassandra/CassandraReloadableKvsConfigTest.java

```diff
@@ -121,6 +121,22 @@ public void ifInstallServersEmpty_resolvesToRuntimeConfig() {
         assertThat(reloadableConfig.concurrentGetRangesThreadPoolSize()).isEqualTo(84);
     }
 
+    @Test
+    public void ifInstallServersEmpty_resolvesToRuntimeConfigForRangesConcurrency() {
```

Comment:
This needs to be tested for all the cases.

Instead of new tests, just add assertions to the existing tests.